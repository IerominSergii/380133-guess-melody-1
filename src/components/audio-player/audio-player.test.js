import React from "react";
import renderer from "react-test-renderer";
import AudioPlayer from "./audio-player";

it(`should render AudioPlayer properly`, () => {
  const component = renderer
    .create(
        <AudioPlayer
          isPlaying={false}
          src={`path.mp3`}
          onPlayButtonClick={jest.fn()}
        />,
        {
          createNodeMock: () => {
            return {};
          }
        }
    )
    .toJSON();

  expect(component).toMatchSnapshot();
});
