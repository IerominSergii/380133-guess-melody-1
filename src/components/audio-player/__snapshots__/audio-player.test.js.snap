// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AudioPlayer properly 1`] = `
<div
  className="game__track"
>
  <button
    className="track__button track__button--play"
    disabled={true}
    onClick={[Function]}
    type="button"
  />
  <div
    className="track__status"
  >
    <audio />
  </div>
</div>
`;
